docker-compose.yml

```diff
@@ -582,15 +582,41 @@ services:
         max-size: 20m
         max-file: '5'
 
+  acl-redis:
+    image: dojot/redis:6.0.4-alpine3.11
+    restart: always
+    volumes:
+      - acl-redis-volume:/data
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  certificate-acl:
+    image: dojot/certificate-acl:development
```

Comment:
the img actually is dojot/dojot-acl

Reply:
maybe we can switch to the acl-certificate in github actions (yml)

Reply:
I only realized why I put the acl in the version I'm working on, which is for 0.8

Reply:
https://github.com/dojot/dojot/pull/2132